src/MathCaptcha.php

```diff
@@ -27,7 +36,10 @@ public function __construct(SessionManager $session = null)
      */
     public function label()
     {
-        return sprintf("%d + %d", $this->getMathFirstOperator(), $this->getMathSecondOperator());
+        if($this->getMathFirstOperator() > $this->getMathSecondOperator())
```

Comment:
Just some tips: I know php develops are kinda lazy, but better still use  {}, even if it's just one line of code.